components/helmdemo/packagespec.yaml

```diff
@@ -1,20 +1,26 @@
 executors:
   - resourceRef:
       resource:
-        #name: toiimage
         name: toiexecutor
       referencePath:
         - name: installer
     config: (( read("helmconfig.yaml") ))
 credentials:
   target:
     description: Kubeconfig for installation
+
+# template used for generating the values file passed to helm
+# merges the values from executors/config/values
+# default helm values can be provided here. Do not forget merging the client
+# parameters. Otherwise no client values can be provided.
 configTemplate:
   namespace: default
   release: echoserver
+  replicaCount: 1
+  <<: (( merge ))     # merge all user provided values with configured defaults
 configScheme:
   type: object
-  additionalProperties: false
+  additionalProperties: true
```

Comment:
Maybe you should adjust the scheme to directly reflect the possible structure to get validation errors in case of using unsupported fields or typos.